Trying to build targeting Mac Catalyst which requires nightly but a dependency fails

If the use of mini_paste isn't too performance critical perhaps moving to paste would make sense since mini_paste hasn't been updated in a couple years.

```
   Compiling mini_paste v0.1.11
error[E0432]: unresolved imports `proc_macro::item`, `proc_macro::__expr_hack__`
  --> /Users/sam/.cargo/registry/src/github.com-1ecc6299db9ec823/mini_paste-0.1.11/src/lib.rs:11:5
   |
11 |     item as __item__,
   |     ----^^^^^^^^^^^^
   |     |
   |     no `item` in the root
   |     help: a similar name exists in the module: `iter`
12 |     __expr_hack__,
   |     ^^^^^^^^^^^^^ no `__expr_hack__` in the root

For more information about this error, try `rustc --explain E0432`.
error: could not compile `mini_paste` due to previous error
```

That patch does get me past compiling mini_paste (thanks!) but then safer_ffi fails to build:

```
error[E0432]: unresolved imports `proc_macro::ffi_export`, `proc_macro::cfg_headers`
  --> /Users/sam/.cargo/registry/src/github.com-1ecc6299db9ec823/safer-ffi-0.0.7/src/_lib.rs:42:24
   |
42 | pub use ::proc_macro::{ffi_export, cfg_headers};
   |                        ^^^^^^^^^^  ^^^^^^^^^^^ no `cfg_headers` in the root
   |                        |
   |                        no `ffi_export` in the root

error[E0432]: unresolved import `proc_macro::derive_ReprC`
  --> /Users/sam/.cargo/registry/src/github.com-1ecc6299db9ec823/safer-ffi-0.0.7/src/_lib.rs:52:13
   |
52 |     pub use ::proc_macro::derive_ReprC;
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^ no `derive_ReprC` in the root

error[E0432]: unresolved import `proc_macro::_proc_macro_hack_c_str`
  --> /Users/sam/.cargo/registry/src/github.com-1ecc6299db9ec823/safer-ffi-0.0.7/src/_lib.rs:49:15
   |
49 |     pub use ::proc_macro::c_str as c;
   |               ^^^^^^^^^^^^^^^^^ no `_proc_macro_hack_c_str` in the root

error[E0432]: unresolved import `proc_macro::derive_ReprC`
  --> /Users/sam/.cargo/registry/src/github.com-1ecc6299db9ec823/safer-ffi-0.0.7/src/layout/_mod.rs:17:9
   |
17 |         derive_ReprC,
   |         ^^^^^^^^^^^^ no `derive_ReprC` in the root
```

BTW, thanks for releasing this library. It made using Rust on iOS so much easier. (I'm working on this: https://github.com/spullara/granne_c)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -4,7 +4,7 @@ path = "src/_lib.rs"
 [package]
 name = "safer-ffi"
 # Keep in sync with `[dependencies.proc_macro]` and `src/proc_macro/Cargo.toml`
-version = "0.0.8"
+version = "0.0.9"
 authors = ["Daniel Henry-Mantilla <daniel.henry.mantilla@gmail.com>"]
 edition = "2018"
 
@@ -59,10 +59,9 @@ require_unsafe_in_body = { version = "0.2.1", optional = true }
 inventory = { version = "0.1.6", optional = true }
 proc-macro-hack = { version = "0.5.15", optional = true }
 
-[dependencies.proc_macro]
-package = "safer_ffi-proc_macro"
+[dependencies.safer_ffi-proc_macro]
 path = "src/proc_macro"
-version = "0.0.8"
+version = "0.0.9"
 
 [dependencies.uninit]
 optional = true

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -3,7 +3,7 @@ path = "src/_lib.rs"
 
 [package]
 name = "safer-ffi"
-# Keep in sync with `[dependencies.proc_macro]` and `src/proc_macro/Cargo.toml`
+# Keep in sync with `[dependencies.safer_ffi-proc_macro]` and `src/proc_macro/Cargo.toml`
 version = "0.0.9"
 authors = ["Daniel Henry-Mantilla <daniel.henry.mantilla@gmail.com>"]
 edition = "2018"
@@ -29,20 +29,20 @@ std = [ "alloc" ]
 
 derives = ["proc_macros"] # alias
 proc_macros = [
-    "proc_macro/proc_macros",
+    "safer_ffi-proc_macro/proc_macros",
     "require_unsafe_in_body",
     "proc-macro-hack",
 ]
 debug_proc_macros = [
     "proc_macros",
-    "proc_macro/verbose-expansions",
+    "safer_ffi-proc_macro/verbose-expansions",
 ]
 
 out-refs = ["uninit"]
 
 headers = [
     "inventory",
-    "proc_macro/headers",
+    "safer_ffi-proc_macro/headers",
     "std",
 ]
 

--- a/src/_lib.rs
+++ b/src/_lib.rs
@@ -38,18 +38,18 @@ extern crate paste;
 mod __utils__;
 use __utils__ as utils;
 
-extern crate proc_macro;
-pub use ::proc_macro::{ffi_export, cfg_headers};
+extern crate safer_ffi_proc_macro;
+pub use ::safer_ffi_proc_macro::{ffi_export, cfg_headers};
 cfg_proc_macros! {
     #[::proc_macro_hack::proc_macro_hack]
     /// Creates a compile-time checked [`char_p::Ref`]`<'static>` out of a
     /// string literal.
     ///
     /// [`char_p::Ref`]: `crate::prelude::char_p::Ref`
-    pub use ::proc_macro::c_str as c;
+    pub use ::safer_ffi_proc_macro::c_str as c;
 
     #[doc(inline)]
-    pub use ::proc_macro::derive_ReprC;
+    pub use ::safer_ffi_proc_macro::derive_ReprC;
 }
 
 #[macro_use]

--- a/src/layout/_mod.rs
+++ b/src/layout/_mod.rs
@@ -13,7 +13,7 @@ mod macros;
 pub use crate::{from_CType_impl_ReprC, ReprC, CType};
 
 cfg_proc_macros! {
-    pub use ::proc_macro::{
+    pub use ::safer_ffi_proc_macro::{
         derive_ReprC,
     };
 }

--- a/src/proc_macro/Cargo.toml
+++ b/src/proc_macro/Cargo.toml
@@ -5,7 +5,7 @@ proc-macro = true
 [package]
 name = "safer_ffi-proc_macro"
 # Keep in sync with `/Cargo.toml`
-version = "0.0.8"
+version = "0.0.9"
 authors = ["Daniel Henry-Mantilla <daniel.henry.mantilla@gmail.com>"]
 edition = "2018"
 

